src/lib/storage/fixed_size_attribute_vector.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+// the linter wants this to be above everything else
+#include <types.hpp>
```

Comment:
Because you make it look like a system header: https://stackoverflow.com/questions/3162030/difference-between-angle-bracket-and-double-quotes-while-including-heade?noredirect=1&lq=1